src/EditorFeatures/Test2/EditorServicesTest2.Next.vbproj

```diff
@@ -10,7 +10,7 @@
     <OptionStrict>Off</OptionStrict>
     <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
     <TargetFrameworkProfile />
-    <RoslynProjectType>UnitTest</RoslynProjectType>
```

Comment:
You will need a Roslyn.sln update for this.  Right now, building Roslyn.sln isn't going to build this...

Not having it is also the reason for your ETA failure.